docs/layouts/shortcodes/build_table.html

```diff
@@ -0,0 +1,37 @@
+<style type="text/css">
```

Comment:
I think this should probably be a list rather than a table based on https://developers.google.com/style/tables